footprints/main/models.py

```diff
@@ -509,8 +516,8 @@ class Person(models.Model):
                                       related_name="death_date",
                                       on_delete=models.CASCADE)
 
-    gender = models.CharField(
-        max_length=1, choices=GENDER_CHOICES, default=UNASSIGNED)
+    gender = models.CharField(max_length=1, choices=GENDER_CHOICES,
```

Comment:
Hi @aaronhpriven, looking at this change, is not a good practice here to use the `choices=GENDER_CHOICES`.  The reason is because `GENDER_CHOICES` will iterate through your enum to perform the `gender.name.capitalize()`.  There is a performance factor that should be considered here.

If using capitalize values in the frontend, is better to do that on the CSS rather than a conversion here.